src/callAPIMethod.js

```diff
@@ -20,21 +20,29 @@ export default function callAPI(
     methodOptions = {}
 ) {
 
-    const { path=[], query={}, options={}, method='json' } = methodOptions;
+    const { path=[], query={}, options={}, method='json', headers, body } = methodOptions;
```

Comment:
@TimeRaider I realized we have the similar options in `methodOptions`. I mean `method` and `options.method`. And now when we can define `header` and `body` two ways it's confusing.

Little example:
```
{
    path: '/user',
    options: {
        method: 'POST',
        body: 'something'
    },
    method: 'json',
    body: 'something'  // some kind of alias for options.body 
}
```
Maybe it would be better to rename `method` -> `type`.
I don't think you have to do it right now. Just think about this.